source/agora/common/Task.d

```diff
@@ -51,4 +58,30 @@ public class TaskManager
         static import vibe.core.core;
         vibe.core.core.sleep(dur);
     }
+
+    /***************************************************************************
+
+        Run an asynchronous task after a given time.
+
+        The task will first run after the given `timeout`, and
+        can either repeat or run only once (the default).
+        Use the `setTimer` vibe.d's
+
+        Params:
+            timeout = Determines the minimum amount of time that elapses before
+                the timer fires.
+            dg = This delegate will be called when the timer fires.
+            periodic = Speficies if the timer fires repeatedly or only once.
+
+    ***************************************************************************/
+
+    public void timer (Duration timeout, void delegate() dg,
```

Comment:
Since both functions are called `setTimer`, why change the name ?

Reply:
I wanted to avoid confusion with the core of vibe.d.

Reply:
But we already have this for `runTask` and `sleep`

Reply:
Yes, I apply your opinion.